project_W/app.py

```diff
@@ -216,7 +231,165 @@ def invalidateAllTokens():
 
         return jsonify(msg="You have successfully been logged out accross all your devices"), 200
 
+    @app.post("/api/jobs/submit")
+    @jwt_required()
+    def submitJob():
```

Comment:
```suggestion
    @jwt_required()
    @activatedRequired
    def submitJob():
```
This route should only work if your user account has been activated. Maybe also /api/runners/create, although that one is currently for admins only anyway. Maybe also add a test case for this?